tests/config/default_2022Q4.json

```diff
@@ -0,0 +1,15 @@
+{
+  "holdingsDate": "2022Q4",
+  "pactaDataURL": "https://pactadatadev.blob.core.windows.net/pacta-data-webapp/2022Q4/2022Q4_20240426T113151Z",
+  "benchmarksURL": "https://pactadatadev.blob.core.windows.net/benchmarks-webapp/2022Q4/2022Q4_20240529T002407Z",
```

Comment:
NB (probably make a new issue): I wonder if we can define this centrally in one place? Seems like it would be very easy for this to get out of sync (across all configs and all `workflow.pacta.*` repos).

Reply:
See #50

Reply:
#50 catches that nicely.